src/state-mutation.js

```diff
@@ -93,7 +93,12 @@ export const removeEntityFromState = (state, entity) => {
       curVal => curVal.filter(l => l.get('id') !== entity.get('id'))
     );
 
-    entity.get('relationships').forEach(relationship => {
```

Comment:
I was getting something along the lines of `Cannot read property of 'forEach' of null/undefined`

The previous code had assumed that`entity.get('relationships')`  would always be present to iterate over. It seems like we'd want to handle the no relationships case like was done on `L#67`